.github/workflows/pytest-workflow.yml

```diff
@@ -56,6 +56,8 @@ jobs:
           #   tags: validation_checks
           # - profile: "conda"
           #   tags: vep
+          # - profile: "conda"
+          #  tags: sentieon/bwamem
```

Comment:
No sentieon tests should be executed with conda - why only mention `sentieon/bwamem` there?
```suggestion
          #  tags: sentieon
```

Reply:
these are commented out, so won't really matter...

Reply:
yeah, I could tell that they were commented out ;-) Not important right now, but why are those comments even there?

Reply:
We need to improve the CI